auto-str.c

```diff
@@ -30,6 +30,11 @@ char **argv;
   puts("[] = \"\\\n");
 
   while (ch = *value++) {
+    if ((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z') || ch == '/' || ch == '_' || ch == '-' || ch == '.') {
```

Comment:
I like where this is going. How about extracting `is_legible()` and instead of `continue`, `if (is_legible(ch)) { //print verbatim } else { //octal it up }` ?

Reply:
use isalnum?

Reply:
```is_alpha()``` maybe. I decided against numbers because it becomes hard to read a number after an octal literal. I used continue so I don't have to change the other lines, but I can do that of course.

Reply:
According to the documentation ```isalpha()``` is locale dependent, which is something I would like to avoid.

Reply:
We don't use setlocale, so this is fine for the C locale.

Reply:
I'm unsure what happens if that program is run in an environment which has a locale set.

Reply:
> On startup of the main program, the portable "C" locale is selected as default.